pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
 
     <artifactId>alfresco-greenmail</artifactId>
-    <version>6.5-SNAPSHOT</version>
+    <version>6.2-SNAPSHOT</version>
```

Comment:
Usually that kind of changes need to be separate from the main PR.

Reply:
Sure, reverted it back to `6.5-SNAPSHOT`.